src/test/java/guitests/DialogMessageTests.java

```diff
@@ -40,6 +40,25 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
     }
 
     @Test
+    public void showYesNoConfirmationDialog_tryYesAndNo_getCorrectValue() throws ExecutionException, InterruptedException {
+        FutureTask<Boolean> yesTask = new FutureTask<>(() ->
+                DialogMessage.showYesNoConfirmationDialog("Confirm", "Confirm Header", "Confirm Message", "yEs", "nO"));
+        Platform.runLater(yesTask);
+        waitUntilNodeAppears(hasText("Confirm Header"));
+        assertNodeExists(hasText("Confirm Header"));
+        clickOn("yEs");
```

Comment:
I think it also tests that the button text are not hardcoded as `Yes` and `No` but can vary according to the arguments?